pkg/build/api/types.go

```diff
@@ -59,6 +59,10 @@ type BuildSpec struct {
 	// Compute resource requirements to execute the build
 	Resources kapi.ResourceRequirements
 
+	// PostCommit is a build hook executed after the build output image is
+	// committed, before it is pushed to a registry.
+	PostCommit BuildPostCommitSpec
```

Comment:
I don't really see that being likely (adding more hooks), and if they are, they wouldn't be the 80% case anyway.  However, this is fine for now.